### DISCUSSION TITLE
Handle M2M Interface when more than Page Count

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3021] [Feature Requests]
If you have more items related than a M2M interface can handle... we need a way to "show all". Perhaps after the first "page" of results, we show a button that opens a modal with the Layout view to browse all.

eg: `customers.invoices` or `iot_devices.events`

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
